addons/website_slides/static/src/xml/activity.xml

```diff
@@ -19,4 +19,32 @@
             </t>
         </t>
     </t>
+
+    <t t-name="slides.Slide" owl="1">
+        <div t-if="slide" class="o_Slide">
+            <button
+                role="button"
+                class="o_Activity_toolButton o_Activity_GrantAccessButton btn btn-link"
+                t-on-click="_onGrantAccess"
+            >
```

Comment:
keep single line for html tags & attributes, it eases readability of the flow of the template